### PR TITLE
[TDL-36, TDL-41] Loyalty history improvements

### PR DESCRIPTION
**Pull request by ardier16**
+ display total fee in the attributes
+ display operation summary in the attributes
+ move movements module translation filters to a mixin
+ display only valuable fees on the table
- remove fee header from the table
- remove unnecessary reconciliation scheme modules

### CODE REVIEW

File: src/vue/modules/movements-history/components/effect-viewers/balance-changed.vue [hunk 1]
```diff
@@ -8,28 +8,47 @@
...
+      v-if="totalFee.value > 0"
+      class="attributes-viewer__table-row"
+    >
+      <td class="attributes-viewer__table-cell">
+        {{ 'movements-history.total-fee-lbl' | globalize }}
+      </td>
+      <td class="attributes-viewer__table-cell">
+        {{ totalFee | formatMoney }}
+      </td>
+    </tr>
```
Comment (max-nes) [inline src/vue/modules/movements-history/components/effect-viewers/balance-changed.vue]: all fee related rows should be shown if totalFee > 0, do not hide them one-by-one. so we provide the user detailed information of what kind of fees were charged, what were not.

File: src/vue/modules/movements-history/components/effect-viewers/particular-balance-changed.vue [hunk 2]
```diff
@@ -8,28 +8,47 @@
...
+      v-if="totalFee.value > 0"
+      class="attributes-viewer__table-row"
+    >
+      <td class="attributes-viewer__table-cell">
+        {{ 'movements-history.total-fee-lbl' | globalize }}
+      </td>
+      <td class="attributes-viewer__table-cell">
+        {{ totalFee | formatMoney }}
+      </td>
+    </tr>
```
Comment (max-nes) [inline src/vue/modules/movements-history/components/effect-viewers/particular-balance-changed.vue]: the same here

File: src/vue/modules/movements-history/components/operation-viewers/payment.vue [hunk 3]
```diff
@@ -1,30 +1,42 @@
...
       </td>
       <td class="attributes-viewer__table-cell">
         {{ destinationFixedFee | formatMoney }}
       </td>
     </tr>
-    <tr class="attributes-viewer__table-row">
+    <tr
+      v-if="destinationCalculatedPercentFee.value > 0"
+      class="attributes-viewer__table-row"
+    >
```
Comment (max-nes) [inline src/vue/modules/movements-history/components/operation-viewers/payment.vue]: the same here.

File: CHANGELOG.md [hunk 4]
```diff
@@ -7,6 +7,23 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
...
+### Added
+- Displaying total fee in the movement attributes
+- Displaying operation summary in the movement attributes 
+
+### Changed
+- Moved movements module translation filters to a mixin
+- Now displaying only valuable fees in the movement attributes
+
+### Removed
+- Fee header from the movements table
```
Comment (max-nes) [inline CHANGELOG.md]: - "Fee" column from movements table

File: CHANGELOG.md [hunk 5]
```diff
@@ -7,6 +7,23 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
...
 for further information about branching and tagging conventions.
 
+## [Unreleased]
+### Added
+- Displaying total fee in the movement attributes
+- Displaying operation summary in the movement attributes 
+
+### Changed
+- Moved movements module translation filters to a mixin
+- Now displaying only valuable fees in the movement attributes
```
Comment (max-nes) [inline CHANGELOG.md]: After the changes below this row may change

File: src/vue/modules/movements-history/components/effect-viewers/balance-changed.vue (hunk 1 above)
Comment (ardier16) [reply src/vue/modules/movements-history/components/effect-viewers/balance-changed.vue]: Updated

File: src/vue/modules/movements-history/components/effect-viewers/particular-balance-changed.vue (hunk 2 above)
Comment (ardier16) [reply src/vue/modules/movements-history/components/effect-viewers/particular-balance-changed.vue]: Updated

File: src/vue/modules/movements-history/components/operation-viewers/payment.vue (hunk 3 above)
Comment (ardier16) [reply src/vue/modules/movements-history/components/operation-viewers/payment.vue]: Updated

File: CHANGELOG.md (hunk 4 above)
Comment (ardier16) [reply CHANGELOG.md]: Renamed

File: CHANGELOG.md (hunk 5 above)
Comment (ardier16) [reply CHANGELOG.md]: Updated

**Pull request merged**